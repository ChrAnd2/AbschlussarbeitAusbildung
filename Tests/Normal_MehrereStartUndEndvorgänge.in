//******************************************
//+ Normal_MehrereStartUndEndvorgänge
//******************************************
//Vorgangsnummer; Vorgangsbezeichnung; Dauer; Vorgänger; Nachfolger
1; A; 1; -; 3;
2; B; 1; -; 4,5
3; C; 2; 1; 6,7
4; D; 4; 2; 7
5; E; 3; 2; 8,9
6; F; 1; 3; -
7; G; 5; 3,4; -
8; H; 8; 5; 10
9; I; 2; 5; 10
10; J; 3; 8,9; -