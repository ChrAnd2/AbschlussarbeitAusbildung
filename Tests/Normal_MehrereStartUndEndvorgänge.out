Normal_MehrereStartUndEndvorgänge

Vorgangsnummer; Vorgangsbezeichnung; D; FAZ; FEZ; SAZ; SEZ; GP; FP
1; A; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0
2; B; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0
3; C; 2.0; 1.0; 3.0; 1.0; 3.0; 0.0; 0.0
4; D; 4.0; 1.0; 5.0; 1.0; 5.0; 0.0; 0.0
5; E; 3.0; 1.0; 4.0; 1.0; 4.0; 0.0; 0.0
6; F; 1.0; 3.0; 4.0; 3.0; 4.0; 0.0; 0.0
7; G; 5.0; 5.0; 10.0; 5.0; 10.0; 0.0; 0.0
8; H; 8.0; 4.0; 12.0; 4.0; 12.0; 0.0; 0.0
9; I; 2.0; 4.0; 6.0; 10.0; 12.0; 6.0; 6.0
10; J; 3.0; 12.0; 15.0; 12.0; 15.0; 0.0; 0.0

Anfangsvorgang: 1,2
Endvorgang: 6,7,10
Maximale Gesamtdauer (da mehrere Endzustaende): 15.0

Kritische Pfade:
1. 1->3->6
2. 1->3->7
3. 2->4->7
4. 2->5->8->10
